### PR TITLE
[deps #396][#95] Integrate matomo web analytics

### LINKED ISSUE #95: Track visits and website usage (matamo or google analytics or something else)
As a site owner/developer/analysts, I want to know how many users and on what platforms are using the site. Are the action flows working or are there issues?

### PR DESCRIPTION
**Pull request by fenekku**
Closes #95

### CODE REVIEW

File: cd2h_repo_project/modules/theme/templates/cd2hrepo_theme/trackingcode.html [hunk 1]
```diff
@@ -1 +1,17 @@
...
+  (function() {
+    var u="https://{{config.ANALYTICS_URL}}/";
+    _paq.push(['setTrackerUrl', u+'matomo.php']);
+    _paq.push(['setSiteId', '{{config.ANALYTICS_SITEID}}']);
+    var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+    g.type='text/javascript'; g.async=true; g.defer=true; g.src='//cdn.matomo.cloud/{{config.ANALYTICS_URL}}/matomo.js'; s.parentNode.insertBefore(g,s);
+  })();
+</script>
+<!-- End Matomo Code -->
+{%- endif -%}
```
Comment (sharpattack) [inline cd2h_repo_project/modules/theme/templates/cd2hrepo_theme/trackingcode.html]: Can you add a link to the matomo documentation for this? Either in the code or as a comment in the issue is fine. I found the set up for most of this, but can not find any reference to the cdn, "//cdn.matomo.cloud/{{config.ANALYTICS_URL}}/matomo.js", can you point me to it?

Comment (carsonicator) [reply cd2h_repo_project/modules/theme/templates/cd2hrepo_theme/trackingcode.html]: Here are the [docs](https://matomo.org/docs/) and [FAQs](https://matomo.org/faq/).

Comment (fenekku) [reply cd2h_repo_project/modules/theme/templates/cd2hrepo_theme/trackingcode.html]: Added info to indicate where the tracking code is found.

Comment (sharpattack) [reply cd2h_repo_project/modules/theme/templates/cd2hrepo_theme/trackingcode.html]: Thank you.

**Comment by fenekku** [comment]
Re-test this please, Jenkins

**Pull request merged**